node.gypi

```diff
@@ -240,12 +294,18 @@
                   '-Wl,-force_load,<(PRODUCT_DIR)/<(OPENSSL_PRODUCT)',
                 ],
               },
+              'msvs_settings': {
+                'VCLinkerTool': {
+                  'AdditionalOptions': [
+                    '/WHOLEARCHIVE:<(PRODUCT_DIR)\\lib\\<(OPENSSL_PRODUCT)',
+                  ],
+                },
+              },
               'conditions': [
                 ['OS in "linux freebsd" and node_shared=="false"', {
                   'ldflags': [
                     '-Wl,--whole-archive,'
-                        '<(OBJ_DIR)/deps/openssl/'
-                        '<(OPENSSL_PRODUCT)',
+                    '<(OBJ_DIR)/deps/openssl/<(OPENSSL_PRODUCT)',
```

Comment:
Can you indent here?  It's a line (and string) continuation.

Reply:
good catch and done